index.d.ts

```diff
@@ -54,6 +54,13 @@ declare module 'fastify' {
       }
     ) => SwaggerSchema.Spec;
   }
+  
+  interface RouteSchema {
+    description?: string;
+    tags?: string[];
+    summary?: string;
+    security?: any[];
```

Comment:
This shouldn't be any; what are the actual objects types of the `security` list?

Reply:
It is being kept as ANY, to accommodate wide variety of use cases. It can be a JWT token or some specific name that is specific to the implementation. I think not keeping it tight to a specific we are opening the possibilities to a wide variety of use cases. Everyone can implement it in their own way. Let me know your thoughts, we can also delete it. if not being applied here.

Reply:
I am new to it so I might be wrong, but shouldn't all this stuff be based on this?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9d9c2ff7419b5e05bf91b29f765beb8d146ee987/types/swagger-schema-official/index.d.ts

Reply:
Yes if this type could infer the values provided by swagger that would be best